src/Idris/REPL/Common.idr

```diff
@@ -50,7 +51,9 @@ printWithStatus : {auto o : Ref ROpts REPLOpts} ->
 printWithStatus render msg
   = do opts <- get ROpts
        case idemode opts of
-         REPL _ => coreLift $ putStrLn !(render msg)
```

Comment:
How about instead of the change you have here deferring to `iputStrLn` (for the `REPL _` case) like you do below in `emitProblem`? That would require an auto implicit ref to Defs here, but maybe it's worth only needing to write `coreLift $ putStrLn !(render msg)` once.

Reply:
I think I actually tried that. But they have different signatures in addition to the `Defs` you mentioned.
Also, the two occurrences of `putStrLn` are far from being the only two in the codebase, unfortunately.
That said, I don't think that's worth the trouble.